plugin.js

```diff
@@ -130,6 +133,18 @@ function cmifyPlugin (b, opts) {
         next(null, row)
       }
     }, function end (done) {
+
+      const outFile = opts.o || opts.outfile
+
+      if (outFile) {
+        try {
+          mkdirp.sync(path.dirname(outFile))
+          fs.writeFileSync(outFile, cmify.getAllCss())
```

Comment:
Future enhancement; Detect if we're in `--verbose` mode, and if so; `console.log('Wrote to file ' + outFile)` or somesuch.